Jenkinsfile

```diff
@@ -598,7 +598,7 @@ pipeline {
   }
   post {
     cleanup {
-      cleanupAzureResources()
+      //cleanupAzureResources()
```

Comment:
There is an ongoing issue and discussion to renew our azure secret. This step uses the same secret, which is currently expired, so this step fails. In order to make the pipeline green I temporarily disable this step.